### DISCUSSION TITLE
Forever "Connecting to wallet"

### DISCUSSION THREAD

**Discussion by rafaroox** [Chia-Network/chia-blockchain#7895] [Ideas]
I upgraded to 1.2.3. I have tried deleteing all of the files in .chia\mainnet\wallet\db.  I am still stuck.  

Please help.

**Comment by crites** [comment]
I'm having the same problem.

**Reply by ItsMrKnowItAll** [reply]
Same here. In fact I just registered my account here in hopes of finding a solution/assistance. I should retract my comment. I'm not stuck on "connecting to wallet" but rather "not synced" endlessly.

**Comment by loppefaaret** [comment]
it might not be your database that is preventing chia from opening up - if you try and open up chia, and then browse to `C:\Users\<USERNAME>\.chia\mainnet\log\` there is a `debug.log` file, and any `ERROR` or `WARNING` gets reported here - if you can tell me what that is, i might be able to direct you to a solution

**Comment by BrandtH22** [comment]
@rafaroox , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
